### ISSUE TITLE
Add an option to change the amount of steps for a keyboard press

### ISSUE THREAD

**Issue by danielneubert** [MonitorControl/MonitorControl#770]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

First things first — I love the new release of MonitorControl. Great work!

My problem is as little as it can get — I prefer fixed values for my contrast on my external display and regularly have to fiddle around a bit to find exactly 50% (where colors are accurate).

### Describe the solution you'd like

An alternative software solution I used while MonitorControl wasn't ready for the M1, gave the option to control the brightness / contrast in fixed steps (in my case: 10%, 30%, 50%, 70% and 90% — So limit 10%/90% and 20% steps).

Something equivalent would be a perfect solution for my problem.

### Describe alternatives you've considered

Actually I have thought of two different solutions:
1. Adding a Icon for the contrast-layover which shows an indicator for the 50% mark:
  ```txt
   (Contrast Icon)
          ↓
  [███████░░░░░░░░]
  ```
2. Adding a keyboard shortcut to set the contrast "back" to 50% with the accurate colors.

**Comment by waydabber** [comment]
Do you want to increase contrast beyond 50% where colors are not accurate or just want to decrease the contrast?

**Comment by waydabber** [comment]
I see you want. Well, you can limit contrast between 10 and 90 already like this:

<img width="544" alt="Screen Shot 2021-11-04 at 19 32 58" src="https://user-images.githubusercontent.com/37590873/140399201-cdd1603e-ed4d-4e24-a061-5facb5b9dd33.png">

50% is quite easy to hit with the keyboard (the click should be beneath the bottom-most "ray" of the sun OSD. If you enable `Slider snapping`, `Show slider tick marks` and maybe `Show percentages`, you can get even easier navigation with the slider as well.

If you limit the contrast between 10 and 90 then in case of slider snapping 0% should be 10, 25% should be 35, 50% should be 50, 75% snap should be 66 and 100% should be 90.

**Comment by danielneubert** [comment]
Hey there, I already found the limit (min/max) which is already a nice start.

I do want to use both, below 50% (in dark rooms at night) and above 50% for playing dark games.

I already found the slider, which I sometimes use. But I'm more of a keyboard-shortcut guy. 

Maybay I have a bug here, but my layover for the contrast doesn't show the sun, just the one for the brightness does.

<img src="https://user-images.githubusercontent.com/18072841/140406854-6939e92b-e4d1-4d7a-9e87-6efbaeb75f64.png" width="500" style="max-width:536px"/>

Is this as expected?
